R/html-tweak.R

```diff
@@ -375,6 +375,28 @@ navbar_links_haystack <- function(html, pkg, path) {
   haystack[haystack$similar > 0, ]
 }
 
+assess_toc <- function(html) {
```

Comment:
Maybe `trim_toc()`? `assess` sounds a bit too passive; I think you want a strong active verb since you're modifying the first argument.